CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-kms ChangeLog
 
+## 6.0.0 - 2021-04-TBD
+
+### Changed
+- **BREAKING**: Drop support for node 10.
```

Comment:
need an `engines` section in the package.json for node >=12

Reply:
added here c230da4